frontend/lib/src/components/widgets/DataFrame/columns/utils.test.ts

```diff
@@ -281,6 +281,14 @@ describe("formatNumber", () => {
     [123456789, "engineering", "123.457E6"],
     [10, "duration[ns]", "a few seconds"],
     [1234567891234, "duration[ns]", "21 minutes"],
+    [10, "period[ms]", "1970-01-01 00:00:00.010"],
+    [10, "period[s]", "1970-01-01 00:00:10"],
+    [10, "period[h]", "1970-01-01 10:00"],
+    [10, "period[min]", "1970-01-01 00:10"],
```

Comment:
extreme nit: swap the lines because `min` < `h`

Reply:
swapped it 👍